### PR TITLE
dotnet add uses ProjectTypeGuid or DefaultProjectTypeGuid

### PR DESCRIPTION
**Pull request by jgoshi**
This is work to address this issue:
https://github.com/dotnet/cli/issues/5131

Because we are still waiting on a change from the SDK, I have some tests commented out and product code isn't quite finished (we use the CPS guid where we will eventually throw, but this is to maintain the same working behavior until the SDK change is available). These places reference the issue we are tracking.

@piotrpMSFT @livarcocc @krwq @jonsequitur

### CODE REVIEW

File: src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs [hunk 1]
```diff
@@ -111,10 +92,46 @@ private void AddProject(SlnFile slnFile, string projectPath)
                 slnFile.Projects.Add(slnProject);
 
                 Reporter.Output.WriteLine(
-                    string.Format(CommonLocalizableStrings.ProjectAddedToTheSolution, projectPath));
+                    string.Format(CommonLocalizableStrings.ProjectAddedToTheSolution, relativeProjectPath));
             }
         }
 
+        private string GetProjectId(ProjectInstance projectInstance)
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs]: Extension of ProjectInstance?

File: src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs [hunk 2]
```diff
@@ -111,10 +92,46 @@ private void AddProject(SlnFile slnFile, string projectPath)
...
+        {
+            var projectGuidProperty = projectInstance.GetPropertyValue("ProjectGuid");
+            var projectGuid = string.IsNullOrEmpty(projectGuidProperty)
+                ? Guid.NewGuid()
+                : new Guid(projectGuidProperty);
+            return projectGuid.ToString("B").ToUpper();
+        }
+
+        private string GetProjectTypeGuid(ProjectInstance projectInstance)
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs]: Extension to ProjectInstance?

File: src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs (hunk 1 above)
Comment (jgoshi) [inline src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs]: Sure I'll add it here src/dotnet/ProjectInstanceExtensions.cs

**Comment by jgoshi** [comment]
@srivatsn

**Comment by jgoshi** [comment]
test OpenSUSE13.2 x64 Debug Build

**Comment by jgoshi** [comment]
test OpenSUSE13.2 x64 Debug Build

**Pull request merged**